editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser2.tsx

```diff
@@ -30,6 +37,59 @@ import { directory } from '../../../core/model/project-file-utils'
 import { DefaultPackageJson, StoryboardFilePath } from '../../editor/store/editor-state'
 import { createCodeFile } from '../../custom-code/code-file.test-utils'
 import { matchInlineSnapshotBrowser } from '../../../../test/karma-snapshots'
+import { EditorAction } from '../../editor/action-types'
+
+async function getControl(
+  controlTestId: string,
+  renderedDOM: RenderResult,
+): Promise<HTMLInputElement> {
+  return (await renderedDOM.findByTestId(controlTestId)) as HTMLInputElement
+}
+
+async function getControlValue(controlTestId: string, renderedDOM: RenderResult): Promise<string> {
+  const control = await getControl(controlTestId, renderedDOM)
+
+  return control.value
+}
+
+async function setControlValue(
+  controlTestId: string,
+  newValue: string,
+  renderedDOM: RenderResult,
+): Promise<void> {
+  const control = await getControl(controlTestId, renderedDOM)
+
+  await act(async () => {
```

Comment:
this could be a sync act, with a sync arrow callback function inside

like this:
```
act(() => {
```